duckduckgo/iOS
# combine tracker networks

Task/Issue URL: https://app.asana.com/0/392891325557410/490128800115961
Tech Design URL: n/a
CC:

**Description**:

Combines tracker networks at the top level.

* No differentiation between networks and major networks
* Major networks are sorted to the top ordered by percentage of sites on the web
* Trackers without a network are sorted to the bottom
* Trackers without a network show the domain as the network and don't show the domain as sub item (see comments in Asana)
* Question mark icon no longer used (see comments in Asana)
* Updated unit tests commented during Privacy Day event
* Fixed typo in name of `percentageOfPages` variable

**Steps to test this PR**:

1. Visit a tracker heavy site (e.g. cnn, the hill, etc)
1. Open privacy dashboard - now just a single item for tracker networks
1. Drill into trackers networks and confirm it looks correct based on above description.
1. Open score card and confirm major trackers and networks shown separately as before.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
